include/psa/crypto_extra.h

```diff
@@ -1901,6 +1901,342 @@ static inline struct psa_pake_operation_s psa_pake_operation_init( void )
     return( v );
 }
 
+/**
+ * \brief The context for PSA interruptible signing.
```

Comment:
Since we're reasonably confident that this is going into the official interface with little to no modifications, we should use the core headers (`crypto_values.h`, `crypto_types.h`, `crypto_struct.h`, `crypto.h`). `crypto_extra.h` is for Mbed TLS extensions (features that will never become official, or features that might become official one day but that we're likely to change).